system/interface.go

```diff
@@ -41,13 +46,41 @@ func (i *DefInterface) setup() error {
 	}
 	i.iface = iface
 	i.exists = true
+	ghwNetwork, err := ghw.Network()
+	if err != nil {
+		i.exists = false
+		i.err = err
+		return i.err
+	}
+	for _, nic := range ghwNetwork.NICs {
+		if nic.Name == i.name {
+			i.ghwIface = nic
+			break
```

Comment:
If this for any reason doesn't ever match. It causes a panic in the code later on when it tries to reference it.